lib/Chat/Parser/SystemMessage.php

```diff
@@ -698,11 +698,16 @@ protected function getRemoteUser(string $federationId): array {
 	}
 
 	protected function getDisplayName(string $uid): string {
+		$userName = $this->userManager->getDisplayName($uid);
+		if ($userName !== $uid) {
+			return $userName;
+		}
+
+		// Verify that the user actually exists
```

Comment:
Do we actually need this? for users without a display name set specifically this will always hit

Reply:
Well we need to know if the user exists.
Talked already with Carl if the `getDisplayName` method could throw or return null in such cases instead of returning the uid

Reply:
Yeah fixed now